scst-store/backups.md

```diff
@@ -0,0 +1,44 @@
+# Backing up Supply Chain Security Tools – Store data
+
+By default, the metadata store uses a `PersistentVolume` mounted on a Postgres instance, making it a stateful component of TAP. When using the provided Postgres instance, a regular backup strategy should be implemented as part of the disaster recovery plan.
+
+[Velero](https://velero.io/) can be used as a tool to created regular backups. 
+
+Note that backup support for `PersistentVolume` depends on the used `StorageClass` and existing provider plugins.
+
+```bash
+velero install --provider <provider> --bucket <bucket-name> --plugins <plugin-image-location> --secret-file <secrets-file>
+
+# Example for GCP:
+# velero install --provider gcp --bucket <gcs-bucket-name> --plugins velero/velero-plugin-for-gcp:v1.3.0 --secret-file <gcp-json-credentials>
+```
+
+Velero CLI can then be used to create a backup of all the resources in the `metadata-store` namespace, including `PersistentVolumeClaim` and `PersistentVolume`. 
+
+```bash
+velero backup create metadata-store-$(date '+%s') --include-namespaces=metadata-store
+```
+
+Velero CLI can then be used to restore the Store in the same or in a different  cluster. The same namespace can be used to restore, but it may collide with other installations of Supply Chain Security Tools – Store. Furthermore, restoring into the same namespace will restore a fully functional instance of Supply Chain Security Tools – Store, but this instance won't be managed by TAP and will conlfict with future installations.
```

Comment:
1) I didn't think the metadata store doesn't support being installed in a different namespace yet though? 

2) `conlfict` -> `conflict`